src/server/pfs/server/driver.go

```diff
@@ -791,17 +787,14 @@ func (d *driver) propagateBranches(txnCtx *txncontext.TransactionContext, branch
 			if err != nil {
```

Comment:
This comment is intended for the `propagateBranches` function comment (I guess you cannot comment on collapsed lines). We should probably remove the reference to a nil HEAD.